package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^12.0.0",
     "@testing-library/user-event": "^13.2.1",
+    "bootstrap": "^5.1.3",
```

Comment:
Do we really need bootstrap? As we want to be unbiased of any CSS framework like bootstrap, material, etc. as much as possible and encourage hands-on CSS as much.

In a long term, we will take call which one it will be. We may go with something like Tailwind also but subject to discussion. So if you could replace the Cards, Col Rows with simple Divs with flex/grid etc, would be great.

Reply:
Okay, but it'd take me a few more days to convert everything to plain CSS.

If we decide to choose Tailwind in the future, I can add more projects from my learning days.

Reply:
done